Wikipedia/Code/WMFWelcomeAnalyticsViewController.m

```diff
@@ -33,20 +34,42 @@ - (void)viewDidLoad {
         self.toggle.on                                                           = NO;
         [[BITHockeyManager sharedHockeyManager] crashManager].crashManagerStatus = BITCrashManagerStatusAlwaysAsk;
     }
+
+    [self.buttonCaretLeft setTintColor:[UIColor wmf_blueTintColor]];
+    [self.buttonCaretLeft wmf_setButtonType:WMFButtonTypeCaretLeft];
 }
 
-- (IBAction)toggleAnalytics:(id)sender {
-    if ([(UISwitch*)sender isOn]) {
+- (IBAction)toggleAnalytics:(UISwitch*)sender {
+    if ([sender isOn]) {
         [[BITHockeyManager sharedHockeyManager] crashManager].crashManagerStatus = BITCrashManagerStatusAutoSend;
         [[NSUserDefaults standardUserDefaults] wmf_setSendUsageReports:YES];
+        
+        dispatchOnMainQueueAfterDelayInSeconds(1.4, ^{
```

Comment:
is it possible to dismiss before the block fires? if so, we should use `@weakify` here to ensure that popping the VC & its view deallocates it, and prevents this from being called twice.